### PR TITLE
Helm & Readme

### PR DESCRIPTION
**Pull request by nirishah**
HelmCharts and Readme file.

### CODE REVIEW

**Comment by amr-mokhtar** [comment]
Hi @nirishah! Please upload the source files (instead of binaries) so that it can be reviewed.
For the readme document it has be named as `README.md`.. A good example is [this app](https://github.com/open-ness/edgeapps/tree/master/applications/actian-zen).

**Comment by nirishah** [comment]
> 
> 
> Hi @nirishah! Please upload the source files (instead of binaries) so that it can be reviewed.
> For the readme document it has be named as `README.md` and written in [Markdown](https://guides.github.com/features/mastering-markdown/) format, and the Helm chart is a folder that named as `helm-chart`.. A good example is [this app](https://github.com/open-ness/edgeapps/tree/master/applications/actian-zen).

I have uploaded the README.md file. please check.

File: applications/radisys/Helm-Chart/templates/vmrf_statefullVM.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Intel Corporation
```
Comment (amr-mokhtar) [inline applications/radisys/Helm-Chart/templates/vmrf_statefullVM.yaml:2]: Feel free to assign the copyright to the owning corporate

File: applications/radisys/readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+**Radisys Engage Media Server**
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:1]: Binary files should be deleted:
- applications/radisys/Readme.docx
- applications/radisys/mrf_new.tar

File: applications/radisys/readme.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Radisys Corporation ©
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:3]: Correct syntax for copyright is explained in [this doc](https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#license-headers).

File: applications/radisys/readme.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Engage Media Server is a media processing powerhouse for a wide range of
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:5]: Editorial: No need to add new line after each line, it gets ignored by Markdown processor.

File: applications/radisys/readme.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Engage Media Server's extensive real-time collaboration capabilities,
+voice and video quality optimizations, and switched and continuous
+presence modes for HD video conferencing. For mobile conferencing
+services, Engage Media Server media server supports IR.92 audio and
+IR.94 video compliant conferencing capabilities. Support for wide
+variety of HD audio and video codecs allows service providers to offer
+ubiquitous multimedia conferencing experiences with automatic
+transcoding across TDM, VoIP, VoLTE, VoNR, VoWiFi, and WebRTC endpoints.
+
+  **Virtual Platforms**               **Minimum**
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:28]: This table will not show correctly unless edited in markdown syntax:

```
|        **Virtual Platforms**        | **Minimum** |
|-------------------------------------|-------------|
| Classification (Filtered Port 80)   | NA          |
| ...                                 | ...         |
...
```

File: applications/radisys/readme.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  Memory (RAM)                        4 GB
+  Storage (System minimum)            30 GB
+  Storage (Content maximum)           
+  Storage Performance                 
+  Host OS                             CentOS 7.x
+  Network                             1G / 10G
+
+**How to Install the vMRF Application**
+
+helm install vmrf \<helmPath\>
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:43]: You can use markdown code block:
~~~
```shell
$ helm install vmrf <helmPath>
```
~~~

File: applications/radisys/readme.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+NAMESPACE: default\
+STATUS: pending-install
+
+## Uninstall vMRF Application
+
+helm uninstall vmrf
+
+## Test Output
+
+\# helm uninstall vmrf\
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:58]:
~~~
```shell
$ helm uninstall vmrf
release "vmrf" uninstalled
```
~~~

File: applications/radisys/readme.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+  Host OS                             CentOS 7.x
+  Network                             1G / 10G
+
+**How to Install the vMRF Application**
+
+helm install vmrf \<helmPath\>
+
+## Test Output
+
+NAME: vmrf\
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:47]:
~~~
```shell
NAME: vmrf
LAST DEPLOYED: Thu Oct 29 23:44:38 2020
NAMESPACE: default
STATUS: pending-install
```
~~~

File: applications/radisys/readme.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+LAST DEPLOYED: Thu Oct 29 23:44:38 2020\
+NAMESPACE: default\
+STATUS: pending-install
+
+## Uninstall vMRF Application
+
+helm uninstall vmrf
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:54]:
~~~
```shell
$ helm uninstall vmrf
```
~~~

File: applications/radisys/readme.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+  Application                         Media server
+  Compute\* (vCores)                  4
+  Memory (RAM)                        4 GB
+  Storage (System minimum)            30 GB
+  Storage (Content maximum)           
+  Storage Performance                 
+  Host OS                             CentOS 7.x
+  Network                             1G / 10G
+
+**How to Install the vMRF Application**
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:41]: Use markdown heading style
~~~
# How to Install the vMRF Application
~~~

File: applications/radisys/readme.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+## Uninstall vMRF Application
+
+helm uninstall vmrf
+
+## Test Output
+
+\# helm uninstall vmrf\
+release \"vmrf\" uninstalled
+
+# **Where to Purchase**
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:61]: Use markdown heading style
~~~
# Where to Purchase
~~~

File: applications/radisys/readme.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+Contact
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:63]:
~~~
# Contact
~~~

File: applications/radisys/readme.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+
+[Radisys Sales Support](https://go.radisys.com/vMRFSolutionlp.html) or
+<https://go.radisys.com/vMRFSolutionlp.html>
+
+Or write mail to sales\@radisys.com
+
+Alternatively, you can call Sales Support: (503) 615-1240
+
+**Resources**
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:72]:
~~~
# Resources
~~~

File: applications/radisys/readme.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+https://hub.radisys.com/mediaenginestream/rsys-computer-vision-whitepaper-101019-c?utm_campaign=cvwp&utm_medium=intelnetworkbuilders
```
Comment (amr-mokhtar) [inline applications/radisys/readme.md:74]: To show as bullet points:
~~~
* https://hub.radisys.com/mediaenginestream/rsys-computer-vision-whitepaper-101019-c?utm_campaign=cvwp&utm_medium=intelnetworkbuilders
* https://hub.radisys.com/mediaenginestream/biometric-authentication-solution-brief?utm_campaign=biometricauthentication&utm_medium=intelnetworkbuilders
* https://hub.radisys.com/mediaenginestream/ds-mediasrvr?utm_campaign=vmrfdatasheet&utm_medium=intelnetworkbuilders
* https://radisys.com
~~~

File: applications/radisys/Helm-Chart/values.yaml [hunk 15]
```diff
@@ -0,0 +1,18 @@
+PersistentVolume:
+  rootfs:
+    size: "32Gi"
+    path: "/var/vd/vol0"
+    imgSrc: "http://10.190.167.100:8081/KVMIMAGE-QBG1G30OC-QBG100-6.3.3.0-142059-600G.img"
```
Comment (amr-mokhtar) [inline applications/radisys/Helm-Chart/values.yaml:5]: Is this a working link?

**Comment by amr-mokhtar** [comment]
Hi @nirishah! Thanks for the updates. The changes look good. Let's try to fix the errors flagged by Travis to allow it passing:
```
Checking Licence Headers
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/readme.md has incorrect licence header
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/values.yaml has incorrect licence header
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/.helmignore has incorrect licence header
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/NOTES.txt has incorrect licence header
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/vmrf_persistentVolume.yaml has incorrect licence header
Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/Chart.yaml has incorrect licence header
```

BTW, the correct copyright header syntax is [here](https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#license-headers).

**Comment by nirishah** [comment]
> 
> 
> Hi @nirishah! Thanks for the updates. The changes look good. Let's try to fix the errors flagged by Travis to allow it passing:
> 
> ```
> Checking Licence Headers
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/readme.md has incorrect licence header
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/values.yaml has incorrect licence header
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/.helmignore has incorrect licence header
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/NOTES.txt has incorrect licence header
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/vmrf_persistentVolume.yaml has incorrect licence header
> Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/Chart.yaml has incorrect licence header
> ```
> 
> BTW, the correct copyright header syntax is [here](https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#license-headers).

Hello @amr-mokhtar, I guess, we need to change the copy right header to suggested format and that is the only license header, right?

**Comment by amr-mokhtar** [comment]
> > Hi @nirishah! Thanks for the updates. The changes look good. Let's try to fix the errors flagged by Travis to allow it passing:
> > ```
> > Checking Licence Headers
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/readme.md has incorrect licence header
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/values.yaml has incorrect licence header
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/.helmignore has incorrect licence header
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/NOTES.txt has incorrect licence header
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/templates/vmrf_persistentVolume.yaml has incorrect licence header
> > Error: File /home/travis/build/open-ness/edgeapps/applications/radisys/Helm-Chart/Chart.yaml has incorrect licence header
> > ```
> > 
> > 
> > BTW, the correct copyright header syntax is [here](https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#license-headers).
> 
> Hello @amr-mokhtar, I guess, we need to change the copy right header to suggested format and that is the only license header, right?

Yes, we should use SPDX style copyright header with the license being Apache-2.0

**Review by amr-mokhtar — APPROVED** [review]
Thanks Radisys team for the contribution. Welcome aboard!!

**Pull request merged**